src/main/resources/google_checks.xml

```diff
@@ -32,6 +32,11 @@
         <property name="eachLine" value="true"/>
     </module>
 
+    <module name="LineLength">
```

Comment:
This change affected google_checks only in this way. No other differences.

Reply:
it is ok.